source/guides/basics/custom-items.md

```diff
@@ -1,26 +1,198 @@
 Fancier Sticks: Making Custom Items (PLACEHOLDER ONLY)
 -----------------------------------
 
-**TODO: Write-up that introduces item script containers, and the general ideas of making custom items, mentioning recipes, name/lore, mechanisms, item flags, etc.**
-
-### Placeholder
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
 
-Until this page is written, you can view the [old tutorial video here](https://one.denizenscript.com/denizen/vids/Custom%20Items).
+### What Are Custom Items
 
-### Sample Script
+Custom items are a way to store a various set of properties onto an item to create a new unique item in the game. Denizen provides the capabilities to save these unique items in item scripts so that they can be reused or uniquely referenced to in other scripts. These were briefly mentioned in [the mechanisms guide](/guides/basics/mechanisms) but this page will go into more detail.
 
-Here's a quick sample of a modern item script.
+### The Basic Structure
 
+To start off with, this is the basic structure of an item script:
 ```dscript_green
-my_item:
+fancy_stick:
     type: item
     material: stick
     display name: <&b><bold>Fancy stick!
     lore:
     - <&7>So fancy.
+    - <&7>Second line.
 ```
+In [the script editor](/guides/first-steps/script-editor), you can simply type `item` and use tab completion to generate the basic template of an item script at any time.
+
+An item script container has the type set to `item`. You also need to give it a valid existing Minecraft material for your item to be based off of; for example here we are using a stick.
+
+While not strictly required, most item script will also have a custom display name and custom lore. The display name works similarly to other container keys, the lore takes in a list as opposed to a single argument. Each line of lore is given on a new line prepended by a `-` <span class="parens">(this is similar, but should not be confused with, how commands look)</span>.
+
+You can get this item in-game by typing `/ex give fancy_stick`
+
+![](images/fancy_stick-basic.png)
+
+### Adding More Properties
+
+There are several other properties that can be added to item scripts.
 
-You can get this item in-game by typing `/ex give my_item`
+As an example this would be how you would add enchantments:
+```dscript_green
+fancy_sword:
+    type: item
+    material: iron_sword
+    display name: <&b><bold>Fancy indestructible sword!
+    enchantments:
+    - DAMAGE_UNDEAD:2
+    - DURABILITY:3
+```
+Note that the enchantment names do differ than the ones that display in game, the names to use can be found on [Spigot's page](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/enchantments/Enchantment.html)
```

Comment:
This is outdated. Minecraft names are now preferred over Bukkit enum names. So <https://minecraft.fandom.com/wiki/Enchanting#Summary_of_enchantments> is a more valid reference to link, if any - but just looking at the creative inventory is arguably easier.